packages/salesforcedx-vscode-lwc/src/commands/forceLightningLwcStart.ts

```diff
@@ -114,28 +114,17 @@ export class ForceLightningLwcStartExecutor extends SfdxCommandletExecutor<{}> {
       }
     });
 
-    // handler errors
-    execution.processExitSubject.subscribe(async exitCode => {
-      DevServerService.instance.clearServerHandler(serverHandler);
-
-      if (!serverStarted && !cancellationToken.isCancellationRequested) {
-        let message = nls.localize('force_lightning_lwc_start_failed');
-
-        // TODO proper exit codes in lwc-dev-server for address in use, auth/org error, etc.
-        if (exitCode === 127) {
-          message = nls.localize('force_lightning_lwc_start_not_found');
-        }
-
-        showError(new Error(message), logName, commandName);
-      } else if (exitCode !== undefined && exitCode !== null && exitCode > 0) {
-        const message = nls.localize(
-          'force_lightning_lwc_start_exited',
-          exitCode
-        );
-        showError(new Error(message), logName, commandName);
+    execution.stderrSubject.subscribe(async data => {
+      if (data && data.toString().includes('Server start up failed')) {
+        this.handleErrors(cancellationToken, serverHandler, 1);
+        progress.complete();
```

Comment:
This piece is needed in order for ProgressNotification to close the open VSCode window. This piece isn't needed for processExitSubject because there is an observer for this event already registered in ProgressNotification.

Reply:
My understanding of using the CLI's `SFDX_JSON_TO_STDOUT: 'true'` environment variable (line 71) was that the command's output will always go through stdout even for error situations. Does that only apply to "core" cli commands and not plugins ?

Reply:
That appears to be the case. From local testing, this definitely doesn't hit the route for stdout.

Reply:
@amphro is the use of `SFDX_JSON_TO_STDOUT: 'true'` exclusive to toolbelt or should plugins also honor that flag ?